app/logic/Mail/OWA/OWAFolder.ts

```diff
@@ -89,19 +89,24 @@ export class OWAFolder extends Folder {
     return true;
   }
 
-  async listMessages(): Promise<ArrayColl<OWAEMail>> {
+  async listMessages() {
     if (!this.dbID) {
       await SQLFolder.save(this);
     }
     await SQLEMail.readAll(this);
```

Comment:
Ah, I see for IMAP you actually refactored that code into `readFolder`. Possibly that belongs into `Folder` rather than just `IMAPFolder`, so that all of the folder classes can benefit from it?

Reply:
Good idea.

Reply:
Done in git commit 92971563